Software/North Star Gen 2/North Star Calibrator/calibrateChessboards.py

```diff
@@ -6,16 +6,15 @@
 
 # CONFIGURATION PARAMETERS
 
-# The number of images to capture before beginning calibration
-numImagesRequired = 10
+
 # The dimension of a single square on the checkerboard in METERS
-checkerboardDimension = 0.0266 # This equates to 26 millimeter wide squares
```

Comment:
Not sure why this value was changed?

Reply:
My bad, part of my checkerboard setup, some people will have different checkerboard sizes

Reply:
ah ok, I'll change it back to .26, that *should* be the size of the checkboard assuming the page is printed properly.